napari/_qt/experimental/render/test_image.py

```diff
@@ -0,0 +1,73 @@
+"""create_text_image: create a PIL image with centered text.
+"""
+import numpy as np
+from PIL import Image, ImageDraw, ImageFont
```

Comment:
Do you think these PIL imports will be needed in the long run?

We only depend on it inadvertently right now because ImageIO depends on it. I guess if it will be there long run we should change this comment then https://github.com/napari/napari/blob/v0.3.8/setup.cfg#L52 so people aren't confused.

Fine to leave for now though

Reply:
It's nice to be able to create "test images" with numbers/letters and lines and stuff. Probably there are other ways to do that? I used PIL because it's common not because I need anything too special.